.github/workflows/windows-msys2-build.yml

```diff
@@ -19,18 +39,38 @@ jobs:
           install: |
             mingw-w64-x86_64-gcc
             mingw-w64-x86_64-autotools
+            mingw-w64-x86_64-sccache
             mingw-w64-x86_64-gtk3
             mingw-w64-x86_64-python-docutils
             patch
             rsync
             unzip
             dos2unix
             mingw-w64-x86_64-nsis
+      # build the GTK bundle, but cache it for a week
+      - name: cache timestamp for GTK-Bundle
+        id: cache_gtk_bundle_timestamp
+        run: echo "timestamp=$(date +%Y-%U)" >> $GITHUB_OUTPUT
```

Comment:
Given we usually release on Sundays, I should use a date that changes on Mondays rather, so we have a chance to test the actual bundle beforehand.
Maybe a way to force rebuilding it as well could be nice (e.g. some job input)